app/javascript/tta_need_app/components/goal_details.jsx

```diff
@@ -67,7 +66,7 @@ class GoalDetails extends PureComponent {
       <div className="box box--bottom-padded">
         <h3>TTA Goal</h3>
         <p className="task-metadata">TTA Goal created by {createdByName} on: {shortDate(createdAt)}</p>
-        {this.title()}
+        <div dangerouslySetInnerHTML={this.title}></div>
```

Comment:
Nice naming there. Is this done so you call out that it shouldn't be used in production as-is?

Reply:
We’ll have to thank the React project for that name. It is to call out that you have to do the work of making sure to avoid XSS attacks.